src/routes/admin.js

```diff
@@ -91,6 +94,17 @@ router.get('/bookmarks.db', isAuthenticated, async (req, res) => {
   res.download(filePath);
 });
 
+router.get('/bookmarks.csv', isAuthenticated, async (req, res) => {
+  const bookmarksDb = req.app.get('bookmarksDb');
+  const bookmarks = await bookmarksDb.getBookmarksForCSVExport();
+  const result = csvStringify(bookmarks);
```

Comment:
I think this needs a tweak

`const result = csvStringify(bookmarks, {quoted: true});`

I had some bookmarks in my database with null values in some columns (from a batch import), and they caused the CSV generation to go awry until I added this option to ensure that they were correctly handled.